Shader log spam

<!--🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟

Thank you for submitting an issue for the relevant topic. Please ensure that you fill in all the required information needed as specified by the template below. Enter text following any "<!-- ✍️\-\-\>" in the template below.

NOTE: If you have other mods installed or you have changed versions; please revert to a clean install and test again with a crash/bug before posting.

🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟-->

## Bug Description

When the world renders the log is filled with the 'same' error message at a rate of 16MB/s

### Did this behavior use to work in the previous version?

No

## Steps to Reproduce (for bugs)

Load a world


## Logs

<!-- 📝
Twitch logs can be found in the installation directory for the Twitch App.  Or click the ... button on SevTech and hit "Open Folder"
ATLauncher logs can be found in the installation directory. Or you can "Open Folder" from the launcher to view the instance.
Then upload the latest/crash logs to PasteBin or Gist. DON'T Upload them to GitHub -->

- Client/Server Log:

https://mclo.gs/pt2DJgV

The repeated part is

ERROR: 0:2: '' :  #version required and missing.
ERROR: 0:5: 'varying' : syntax error: syntax error
	at com.mojang.blaze3d.shaders.Program.m_166612_(Program.java:69) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:classloading,pl:accesstransformer:B}
	at com.mojang.blaze3d.shaders.EffectProgram.m_166588_(EffectProgram.java:41) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:classloading}
	at net.minecraft.client.renderer.EffectInstance.m_172566_(EffectInstance.java:159) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.EffectInstance.<init>(EffectInstance.java:128) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	... 60 more
[16:44:44] [Render thread/WARN] [de.ce.co.cl.re.EffectRenderer/]: Failed to load shader: colorblindness:shaders/post/tritanopia.json
net.minecraft.server.ChainedJsonException: Invalid shaders/program/color_shift.json (mod_resources): Couldn't compile fragment program (mod_resources, colorblindness:color_shift) : ERROR: 0:1: '' :  version '110' is not supported
ERROR: 0:2: '' :  #version required and missing.
ERROR: 0:5: 'varying' : syntax error: syntax error
	at net.minecraft.server.ChainedJsonException.m_135906_(ChainedJsonException.java:48) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:classloading}
	at net.minecraft.client.renderer.EffectInstance.<init>(EffectInstance.java:139) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.PostPass.<init>(PostPass.java:31) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,re:classloading,pl:mixin:APP:color_stages.mixins.json:MixinPostPass,pl:mixin:A}
	at net.minecraft.client.renderer.PostChain.m_110042_(PostChain.java:285) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:runtimedistcleaner:A,re:computing_frames,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.PostChain.m_110030_(PostChain.java:134) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:runtimedistcleaner:A,re:computing_frames,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.PostChain.m_110033_(PostChain.java:86) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:runtimedistcleaner:A,re:computing_frames,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.PostChain.<init>(PostChain.java:54) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:runtimedistcleaner:A,re:computing_frames,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at de.cech12.colorblindness.client.renderer.EffectRenderer.createShaderGroup(EffectRenderer.java:52) ~[colorblindness-1.20.1-2.1.0.0.jar%23475!/:1.20.1-2.1.0.0] {re:classloading}
	at de.cech12.colorblindness.client.renderer.EffectRenderer.makeColorShaders(EffectRenderer.java:84) ~[colorblindness-1.20.1-2.1.0.0.jar%23475!/:1.20.1-2.1.0.0] {re:classloading}
	at de.cech12.colorblindness.client.renderer.EffectRenderer.onRender(EffectRenderer.java:96) ~[colorblindness-1.20.1-2.1.0.0.jar%23475!/:1.20.1-2.1.0.0] {re:classloading}
	at de.cech12.colorblindness.client.renderer.__EffectRenderer_onRender_RenderTickEvent.invoke(.dynamic) ~[colorblindness-1.20.1-2.1.0.0.jar%23475!/:1.20.1-2.1.0.0] {re:classloading,pl:eventbus:B}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.5.jar%23125!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.5.jar%23125!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.5.jar%23125!/:?] {}
	at net.minecraftforge.event.ForgeEventFactory.onRenderTickEnd(ForgeEventFactory.java:910) ~[forge-1.20.1-47.3.0-universal.jar%23723!/:?] {re:classloading}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1148) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:APP:botania_xplat.mixins.json:client.MinecraftAccessor,pl:mixin:APP:mixins.codechickenlib.json:MinecraftMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:embeddiumplus.mixin.json:fps.GpuUsageMixin,pl:mixin:APP:globalpacks.mixins.json:MinecraftMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.client.AccessorMinecraft,pl:mixin:APP:railways-common.mixins.json:conductor_possession.MixinMinecraft,pl:mixin:APP:carryon.mixins.json:MinecraftMixin,pl:mixin:APP:bridgingmod.mixins.json:MinecraftClientMixin,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:controllable.common.mixins.json:client.MinecraftMixin,pl:mixin:APP:controllable.mixins.json:client.ForgeMinecraftMixin,pl:mixin:APP:ichunutil.mixins.json:client.MinecraftMixin,pl:mixin:APP:ae2.mixins.json:PickColorMixin,pl:mixin:APP:moonlight-common.mixins.json:MinecraftMixin,pl:mixin:APP:iceberg.mixins.json:MinecraftMixin,pl:mixin:APP:ars_nouveau.mixins.json:light.ClientMixin,pl:mixin:APP:quartz.mixins.json:MinecraftShutdownMixin,pl:mixin:APP:yacl.mixins.json:MinecraftMixin,pl:mixin:APP:rrls.mixins.json:MinecraftClientMixin,pl:mixin:APP:create.mixins.json:client.WindowResizeMixin,pl:mixin:APP:embeddium.mixins.json:core.MinecraftClientMixin,pl:mixin:APP:embeddium.mixins.json:core.render.MinecraftAccessor,pl:mixin:APP:ars_nouveau.mixins.json:camera.MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91320_(Minecraft.java:2117) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:APP:botania_xplat.mixins.json:client.MinecraftAccessor,pl:mixin:APP:mixins.codechickenlib.json:MinecraftMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:embeddiumplus.mixin.json:fps.GpuUsageMixin,pl:mixin:APP:globalpacks.mixins.json:MinecraftMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.client.AccessorMinecraft,pl:mixin:APP:railways-common.mixins.json:conductor_possession.MixinMinecraft,pl:mixin:APP:carryon.mixins.json:MinecraftMixin,pl:mixin:APP:bridgingmod.mixins.json:MinecraftClientMixin,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:controllable.common.mixins.json:client.MinecraftMixin,pl:mixin:APP:controllable.mixins.json:client.ForgeMinecraftMixin,pl:mixin:APP:ichunutil.mixins.json:client.MinecraftMixin,pl:mixin:APP:ae2.mixins.json:PickColorMixin,pl:mixin:APP:moonlight-common.mixins.json:MinecraftMixin,pl:mixin:APP:iceberg.mixins.json:MinecraftMixin,pl:mixin:APP:ars_nouveau.mixins.json:light.ClientMixin,pl:mixin:APP:quartz.mixins.json:MinecraftShutdownMixin,pl:mixin:APP:yacl.mixins.json:MinecraftMixin,pl:mixin:APP:rrls.mixins.json:MinecraftClientMixin,pl:mixin:APP:create.mixins.json:client.WindowResizeMixin,pl:mixin:APP:embeddium.mixins.json:core.MinecraftClientMixin,pl:mixin:APP:embeddium.mixins.json:core.render.MinecraftAccessor,pl:mixin:APP:ars_nouveau.mixins.json:camera.MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.screens.PauseScreen.m_261092_(PauseScreen.java:102) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.multiplayer.chat.report.ReportingContext.m_261157_(ReportingContext.java:53) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:classloading}
	at net.minecraft.client.gui.screens.PauseScreen.lambda$createPauseMenu$6(PauseScreen.java:90) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.components.Button.m_5691_(Button.java:38) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.components.AbstractButton.m_5716_(AbstractButton.java:55) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.components.AbstractWidget.m_6375_(AbstractWidget.java:175) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.components.events.ContainerEventHandler.m_6375_(ContainerEventHandler.java:38) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,re:computing_frames,re:classloading}
	at net.minecraft.client.MouseHandler.m_168084_(MouseHandler.java:92) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:zume.mixins.json:lexforge.MouseHandlerMixin,pl:mixin:APP:ldlib-common.mixins.json:accessor.MouseHandlerAccessor,pl:mixin:APP:balm.mixins.json:MouseHandlerAccessor,pl:mixin:APP:supplementaries-common.mixins.json:MouseHandlerMixin,pl:mixin:APP:mixins.cofhcore.json:MouseHandlerMixin,pl:mixin:APP:railways-common.mixins.json:conductor_possession.MixinMouseHandler,pl:mixin:APP:controllable.common.mixins.json:client.MouseHelperMixin,pl:mixin:APP:ichunutil.mixins.json:client.MouseHandlerMixin,pl:mixin:APP:rrls.mixins.json:compat.RendererKeyboardMouseMixin,pl:mixin:APP:create.mixins.json:accessor.MouseHandlerAccessor,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.screens.Screen.m_96579_(Screen.java:437) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:ScreenAccessor,pl:mixin:APP:mixins.transmog.common.json:ScreenMixin,pl:mixin:APP:controlling.mixins.json:AccessScreen,pl:mixin:APP:refurbished_furniture.common.mixins.json:client.ScreenAccessor,pl:mixin:APP:patchouli_xplat.mixins.json:client.AccessorScreen,pl:mixin:APP:ae2.mixins.json:WrappedGenericStackTooltipModIdMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.MouseHandler.m_91530_(MouseHandler.java:89) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:zume.mixins.json:lexforge.MouseHandlerMixin,pl:mixin:APP:ldlib-common.mixins.json:accessor.MouseHandlerAccessor,pl:mixin:APP:balm.mixins.json:MouseHandlerAccessor,pl:mixin:APP:supplementaries-common.mixins.json:MouseHandlerMixin,pl:mixin:APP:mixins.cofhcore.json:MouseHandlerMixin,pl:mixin:APP:railways-common.mixins.json:conductor_possession.MixinMouseHandler,pl:mixin:APP:controllable.common.mixins.json:client.MouseHelperMixin,pl:mixin:APP:ichunutil.mixins.json:client.MouseHandlerMixin,pl:mixin:APP:rrls.mixins.json:compat.RendererKeyboardMouseMixin,pl:mixin:APP:create.mixins.json:accessor.MouseHandlerAccessor,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.MouseHandler.m_168091_(MouseHandler.java:189) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:zume.mixins.json:lexforge.MouseHandlerMixin,pl:mixin:APP:ldlib-common.mixins.json:accessor.MouseHandlerAccessor,pl:mixin:APP:balm.mixins.json:MouseHandlerAccessor,pl:mixin:APP:supplementaries-common.mixins.json:MouseHandlerMixin,pl:mixin:APP:mixins.cofhcore.json:MouseHandlerMixin,pl:mixin:APP:railways-common.mixins.json:conductor_possession.MixinMouseHandler,pl:mixin:APP:controllable.common.mixins.json:client.MouseHelperMixin,pl:mixin:APP:ichunutil.mixins.json:client.MouseHandlerMixin,pl:mixin:APP:rrls.mixins.json:compat.RendererKeyboardMouseMixin,pl:mixin:APP:create.mixins.json:accessor.MouseHandlerAccessor,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.util.thread.BlockableEventLoop.execute(BlockableEventLoop.java:102) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.client.MouseHandler.m_91565_(MouseHandler.java:188) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:zume.mixins.json:lexforge.MouseHandlerMixin,pl:mixin:APP:ldlib-common.mixins.json:accessor.MouseHandlerAccessor,pl:mixin:APP:balm.mixins.json:MouseHandlerAccessor,pl:mixin:APP:supplementaries-common.mixins.json:MouseHandlerMixin,pl:mixin:APP:mixins.cofhcore.json:MouseHandlerMixin,pl:mixin:APP:railways-common.mixins.json:conductor_possession.MixinMouseHandler,pl:mixin:APP:controllable.common.mixins.json:client.MouseHelperMixin,pl:mixin:APP:ichunutil.mixins.json:client.MouseHandlerMixin,pl:mixin:APP:rrls.mixins.json:compat.RendererKeyboardMouseMixin,pl:mixin:APP:create.mixins.json:accessor.MouseHandlerAccessor,pl:mixin:A,pl:runtimedistcleaner:A}
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43) ~[lwjgl-glfw-3.3.1.jar%2373!/:build 7] {}
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.3.1.jar%2385!/:build 7] {}
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3403) ~[lwjgl-glfw-3.3.1.jar%2373!/:build 7] {re:mixin}
	at com.mojang.blaze3d.systems.RenderSystem.pollEvents(RenderSystem.java:201) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:flywheel.mixins.json:RenderTexturesMixin,pl:mixin:APP:embeddium.mixins.json:workarounds.event_loop.RenderSystemMixin,pl:mixin:A}
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:219) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:flywheel.mixins.json:RenderTexturesMixin,pl:mixin:APP:embeddium.mixins.json:workarounds.event_loop.RenderSystemMixin,pl:mixin:A}
	at com.mojang.blaze3d.platform.Window.m_85435_(Window.java:274) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:embeddiumplus.mixin.json:borderless.BorderlessWindowMixin,pl:mixin:APP:embeddiumplus.mixin.json:borderless.accessors.MainWindowAccessor,pl:mixin:APP:quartz.mixins.json:QuartzStartupMixin,pl:mixin:APP:embeddium.mixins.json:workarounds.context_creation.WindowMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1170) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:APP:botania_xplat.mixins.json:client.MinecraftAccessor,pl:mixin:APP:mixins.codechickenlib.json:MinecraftMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:embeddiumplus.mixin.json:fps.GpuUsageMixin,pl:mixin:APP:globalpacks.mixins.json:MinecraftMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.client.AccessorMinecraft,pl:mixin:APP:railways-common.mixins.json:conductor_possession.MixinMinecraft,pl:mixin:APP:carryon.mixins.json:MinecraftMixin,pl:mixin:APP:bridgingmod.mixins.json:MinecraftClientMixin,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:controllable.common.mixins.json:client.MinecraftMixin,pl:mixin:APP:controllable.mixins.json:client.ForgeMinecraftMixin,pl:mixin:APP:ichunutil.mixins.json:client.MinecraftMixin,pl:mixin:APP:ae2.mixins.json:PickColorMixin,pl:mixin:APP:moonlight-common.mixins.json:MinecraftMixin,pl:mixin:APP:iceberg.mixins.json:MinecraftMixin,pl:mixin:APP:ars_nouveau.mixins.json:light.ClientMixin,pl:mixin:APP:quartz.mixins.json:MinecraftShutdownMixin,pl:mixin:APP:yacl.mixins.json:MinecraftMixin,pl:mixin:APP:rrls.mixins.json:MinecraftClientMixin,pl:mixin:APP:create.mixins.json:client.WindowResizeMixin,pl:mixin:APP:embeddium.mixins.json:core.MinecraftClientMixin,pl:mixin:APP:embeddium.mixins.json:core.render.MinecraftAccessor,pl:mixin:APP:ars_nouveau.mixins.json:camera.MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:718) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:APP:botania_xplat.mixins.json:client.MinecraftAccessor,pl:mixin:APP:mixins.codechickenlib.json:MinecraftMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:embeddiumplus.mixin.json:fps.GpuUsageMixin,pl:mixin:APP:globalpacks.mixins.json:MinecraftMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.client.AccessorMinecraft,pl:mixin:APP:railways-common.mixins.json:conductor_possession.MixinMinecraft,pl:mixin:APP:carryon.mixins.json:MinecraftMixin,pl:mixin:APP:bridgingmod.mixins.json:MinecraftClientMixin,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:controllable.common.mixins.json:client.MinecraftMixin,pl:mixin:APP:controllable.mixins.json:client.ForgeMinecraftMixin,pl:mixin:APP:ichunutil.mixins.json:client.MinecraftMixin,pl:mixin:APP:ae2.mixins.json:PickColorMixin,pl:mixin:APP:moonlight-common.mixins.json:MinecraftMixin,pl:mixin:APP:iceberg.mixins.json:MinecraftMixin,pl:mixin:APP:ars_nouveau.mixins.json:light.ClientMixin,pl:mixin:APP:quartz.mixins.json:MinecraftShutdownMixin,pl:mixin:APP:yacl.mixins.json:MinecraftMixin,pl:mixin:APP:rrls.mixins.json:MinecraftClientMixin,pl:mixin:APP:create.mixins.json:client.WindowResizeMixin,pl:mixin:APP:embeddium.mixins.json:core.MinecraftClientMixin,pl:mixin:APP:embeddium.mixins.json:core.render.MinecraftAccessor,pl:mixin:APP:ars_nouveau.mixins.json:camera.MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:218) ~[minecraft-1.20.1-client.jar:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:flywheel.mixins.json:ClientMainMixin,pl:mixin:APP:smoothboot.mixins.json:client.MainMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111) ~[fmlloader-1.20.1-47.3.0.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.clientService(CommonLaunchHandler.java:99) ~[fmlloader-1.20.1-47.3.0.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$makeService$0(CommonClientLaunchHandler.java:25) ~[fmlloader-1.20.1-47.3.0.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:108) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:67) ~[?:?] {}
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:105) ~[?:?] {}
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) ~[?:?] {}
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) ~[?:?] {}
Caused by: java.io.IOException: Couldn't compile fragment program (mod_resources, colorblindness:color_shift) : ERROR: 0:1: '' :  version '110' is not supported
ERROR: 0:2: '' :  #version required and missing.
ERROR: 0:5: 'varying' : syntax error: syntax error
	at com.mojang.blaze3d.shaders.Program.m_166612_(Program.java:69) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:classloading,pl:accesstransformer:B}
	at com.mojang.blaze3d.shaders.EffectProgram.m_166588_(EffectProgram.java:41) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:classloading}
	at net.minecraft.client.renderer.EffectInstance.m_172566_(EffectInstance.java:159) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.EffectInstance.<init>(EffectInstance.java:128) ~[client-1.20.1-20230612.114412-srg.jar%23718!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	... 60 more



- Crash Log:n/a

## World Information

- Modpack Version world created in:5.0.1
- Additional Content Installed:n/a

## Client Information

- Modpack Version:<!-- ✍️-->
- Java Version:ava is version 17.0.8, using 64 (aarch64) architecture, from Microsoft.
- Launcher Used:Prism Launcher version: 9.1 (official)
- Memory Allocated:8000Mb
- Server/LAN/Single Player:Single
- Optifine Installed:no
- Shaders Enabled:pack default

--- a/src/minecraft/thingpacks/sf5_things/assets/colorblindness/shaders/program/color_shift.fsh
+++ b/src/minecraft/thingpacks/sf5_things/assets/colorblindness/shaders/program/color_shift.fsh
@@ -1,0 +1,24 @@
+#version 150
+
+uniform sampler2D DiffuseSampler;
+
+in vec2 texCoord;
+in vec2 oneTexel;
+
+out vec4 fragColor;
+
+uniform vec3 RedMatrix;
+uniform vec3 GreenMatrix;
+uniform vec3 BlueMatrix;
+
+void main() {
+    vec4 InTexel = texture(DiffuseSampler, texCoord);
+
+    // Color Matrix
+    float RedValue = dot(InTexel.rgb, RedMatrix);
+    float GreenValue = dot(InTexel.rgb, GreenMatrix);
+    float BlueValue = dot(InTexel.rgb, BlueMatrix);
+    vec3 OutColor = vec3(RedValue, GreenValue, BlueValue);
+
+    fragColor = vec4(OutColor, 1.0);
+}

--- a/src/minecraft/thingpacks/sf5_things/assets/colorblindness/shaders/program/color_shift.json
+++ b/src/minecraft/thingpacks/sf5_things/assets/colorblindness/shaders/program/color_shift.json
@@ -1,0 +1,20 @@
+{
+    "blend": {
+        "func": "add",
+        "srcrgb": "one",
+        "dstrgb": "zero"
+    },
+    "vertex": "sobel",
+    "fragment": "colorblindness:color_shift",
+    "attributes": [ "Position" ],
+    "samplers": [
+        { "name": "DiffuseSampler" }
+    ],
+    "uniforms": [
+        { "name": "ProjMat",     "type": "matrix4x4", "count": 16, "values": [ 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0 ] },
+        { "name": "OutSize",     "type": "float",     "count": 2,  "values": [ 1.0, 1.0 ] },
+        { "name": "RedMatrix",   "type": "float",     "count": 3,  "values": [ 1.0, 0.0, 0.0 ] },
+        { "name": "GreenMatrix", "type": "float",     "count": 3,  "values": [ 0.0, 1.0, 0.0 ] },
+        { "name": "BlueMatrix",  "type": "float",     "count": 3,  "values": [ 0.0, 0.0, 1.0 ] }
+    ]
+}

